controller/reportController.js

```diff
@@ -156,9 +156,46 @@ module.exports = {
 
       /** addManage에 등록 */
       const result = await reportService.registerAddCafe(userId, registerAddCafeId);
-      return res.status(statusCode.OK).send(util.success(statusCode.OK,responseMessage.REGISTER_ADD_CAFE_SUCCESS));
+      return res.status(statusCode.OK).send(util.success(statusCode.OK, responseMessage.REGISTER_ADD_CAFE_SUCCESS));
     } catch (error) {
-      return res.status(statusCode.INTERNAL_SERVER_ERROR).send(util.fail(statusCode.INTERNAL_SERVER_ERROR,responseMessage.INTERNAL_SERVER_ERROR));
+      return res.status(statusCode.INTERNAL_SERVER_ERROR).send(util.fail(statusCode.INTERNAL_SERVER_ERROR, responseMessage.INTERNAL_SERVER_ERROR));
+    }
+  },
+  addMenu: async (req, res) => {
+    const userId = req.userIdx;
+    const { cafeId } = req.params;
+    const { menu } = req.body;
+
+    try {
+      if (!userId || !menu) {
+        return res.status(statusCode.BAD_REQUEST).send(util.fail(statusCode.BAD_REQUEST, responseMessage.NULL_VALUE));
+      }
+  
+      if (!cafeId) {
+        return res.status(statusCode.BAD_REQUEST).send(util.fail(statusCode.NOT_FOUND, responseMessage.NOT_EXISTING_CAFE));
+      }
+
+      const existingMenu = await cafeService.readCafeMenu(cafeId);
+      if (existingMenu) {
+        return res.status(statusCode.BAD_REQUEST).send(util.fail(statusCode.BAD_REQUEST, responseMessage.ALREADY_REGISTERED_MENU));
+      }
+
+      /** menu 등록 */
+      var registerCafeMenu = {};
```

Comment:
```suggestion
    const registerCafeMenu = { registerCafeMenuTemp: [] };
```
이렇게 하면 어떠세요??
그리고 var 는 사용하지 않는게 좋습니다 ㅎㅎ

Reply:
아하 그렇군요!!!!! 넵 너무 좋습니당ㅎㅎㅎ 다시 반영해놓을게요!!